.github/workflows/all-builds.yml

```diff
@@ -1,16 +1,16 @@
 name: all-builds
 
-on:
-  schedule:
-    - cron: "0 23 * * *"
-  push:
-    branches: 
-      - '*'
-    tags:
-      - '*'
-
-  pull_request:
-    branches: [ master ]
+on: workflow_dispatch
+  # schedule:
+  #   - cron: "0 23 * * *"
+  # push:
+  #   branches: 
+  #     - '*'
+  #   tags:
+  #     - '*'
+
+  # pull_request:
```

Comment:
Wouldn't we want to build this even on a `push` or a  `pull_request` to master?

Reply:
no, these jobs fail, they build things we don't need and then try and fail to push them to the official dockerhub repo